packages/block-editor/src/components/video-player/index.native.js

```diff
@@ -84,7 +85,10 @@ class Video extends Component {
 	}
 
 	render() {
-		const { isSelected, style } = this.props;
+		const { isSelected, style, forceFullscreen } = this.props;
+		if ( forceFullscreen !== undefined ) {
+			this.forceFullscreen = forceFullscreen;
+		}
```

Comment:
Maybe using a default value is cleaner? In this way we don't need to declare `this.forceFullscreen`
```suggestion
		const { isSelected, style, forceFullscreen = ! this.isIOS } = this.props;
```